FreeRTOS-Plus/Demo/Common/Mqtt_Demo_Helpers/mqtt_demo_helpers.h

```diff
@@ -53,41 +60,49 @@ void vHandleOtherIncomingPacket( MQTTPacketInfo_t * pxPacketInfo,
 /**
  * @brief Close the MQTT connection.
  *
+ * @param[in] pxContext The MQTT context for the MQTT connection to close.
+ *
  * @return pdPASS if DISCONNECT was successfully sent;
  * pdFAIL otherwise.
  */
-BaseType_t xDisconnectMqttSession( void );
+BaseType_t xDisconnectMqttSession( MQTTContext_t * pxContext,
+                                   NetworkContext_t * pxNetContext );
 
 /**
  * @brief Subscribe to a MQTT topic filter.
  *
+ * @param[in] pxContext The MQTT context for the MQTT connection to close.
  * @param[in] pcTopicFilter Pointer to the shadow topic buffer.
  * @param[in] usTopicFilterLength Indicates the length of the shadow
- * topic filter.
+ * topic buffer.
  *
  * @return pdPASS if SUBSCRIBE was successfully sent;
  * pdFAIL otherwise.
  */
-BaseType_t xSubscribeToTopic( const char * pcTopicFilter,
+BaseType_t xSubscribeToTopic( MQTTContext_t * pxContext,
+                              const char * pcTopicFilter,
                               uint16_t usTopicFilterLength );
 
 /**
  * @brief Sends an MQTT UNSUBSCRIBE to unsubscribe from the shadow
  * topic.
  *
+ * @param[in] pxContext The MQTT context for the MQTT connection.
  * @param[in] pcTopicFilter Pointer to the shadow topic buffer.
  * @param[in] usTopicFilterLength Indicates the length of the shadow
- * topic filter.
+ * topic buffer.
```

Comment:
Wasn't this better as filter? The buffer could be larger than the actual filter.

Reply:
I don't remember making this change explicitly...Will revert it back.